Sources/MongoSwift/Operations/RunCommandOperation.swift

```diff
@@ -10,6 +10,10 @@ public struct RunCommandOptions: Encodable {
     /// command that reads.
     public var readPreference: ReadPreference?
 
+    /// Opaque value representing a server to run the command on.
+    /// For usage in tests only.
+    internal var serverId: Int?
```

Comment:
it seems a little confusing to me that this exists on the struct but then gets filled in via the runCommand operation. I could see someone trying to provide an id directly via the options, like
```swift
var opts = RunCommandOptions()
opts.serverId = 123
```

only to have it overwritten when the command is actually being executed. it might be a little clearer to just add the server id directly to the document returned from `encodeOptions`